src/arguments.rs

```diff
@@ -2,7 +2,7 @@ use super::*;
 
 #[derive(StructOpt)]
 pub(crate) struct Arguments {
-  pub(crate) filters: Vec<Filter>,
+  pub(crate) commands: Vec<Command>,
   #[structopt(long)]
   pub(crate) output: Option<PathBuf>,
```

Comment:
Feels weird keeping `output` when we have a `Save` command, and we can easily add a `Print` command that passes stdout to `state::write`. Although it might be necessary for determining what to do when all commands have been applied. What do you think?

Reply:
Agreed, I like the idea of adding a `print` command, and then just getting rid of `Arguments::output` entirely. At that point we don't even need structopt and can just parse the arguments ourselves. I created #51 to track this.

Reply:
Sounds good, we could leave this for another PR